pkg/bidichk/bidichk.go

```diff
@@ -51,12 +51,12 @@ func check(filename string, pos token.Pos, pass *analysis.Pass) error {
 	for name, r := range disallowedRunes {
 		start := 0
 		for {
-			if !bytes.ContainsRune(body[start:], r) {
+			idx := bytes.IndexRune(body[start:], r)
+			if idx == -1 {
 				break
 			}
-			start += bytes.IndexRune(body[start:], r)
-			pass.Reportf(pos+token.Pos(start), "found dangerous unicode character sequence %s", name)
-			start += utf8.RuneLen(r)
+			start += idx + utf8.RuneLen(r)
+			pass.Reportf(pos+token.Pos(idx), "found dangerous unicode character sequence %s", name)
```

Comment:
`token.Pos(idx)` is wrong, because `idx` is the position from the last occurrence and not from the beginning of the file.

Reply:
oh yeah, sorry, last suggestion was incorrect

Reply:
I dropped the commit